k8s/api-claim0-persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  creationTimestamp: null
+  labels:
+    io.kompose.service: api-claim0
+  name: api-claim0
```

Comment:
Change the name to be more meaningful?

Reply:
is "api-pv" okay? maybe do you have a better name?

Reply:
for example, in this pvc, the usage is in https://github.com/guy9050/monocle/blob/k8s_manifest/k8s/api-deployment.yaml#L62 and it holds the config.yaml file, but I just grepped for `api-claim0` and tried to figure out what is the actual usage of the volume

Reply:
@georgettica +1 "config" would be the right name imo. Other deployment resources will need it too.

Reply:
changed